### ISSUE TITLE
Not support unityhub

### ISSUE THREAD

**Issue by Thaina** [Unity-Technologies/vscode-unity-debug#69]
```
UnityDebug: Initializing
UnityDebug: Searching for Unity process 'Unity Editor'
UnityDebug: Multiple targets with name 'Unity Editor' running. Unable to connect
UnityDebug: Found Unity process 'Unity Editor (Unity Hub)' (16816)
UnityDebug: Found Unity process 'Unity Editor (UnityCrashHandler64)' (5280)
UnityDebug: Found Unity process 'Unity Editor (Unity)' (4796)
UnityDebug: Found Unity process 'Unity Editor (Unity Hub)' (18908)
UnityDebug: Disconnected
```

Please add selector

**Comment by dps999** [comment]
I got the same issue after started to use Unity hub.
You can fix it by change the line to
            "name": "Unity Editor (Unity)",

![2018-02-11_19-32-03](https://user-images.githubusercontent.com/1697478/36075603-5282e3b4-0f62-11e8-8044-fb04b28b05e5.png)

**Comment by lukaszunity** [comment]
Fixed in published 1.3.0 version.
